### PR TITLE
fix: refresh prices after changing the oracle order

### PR DESCRIPTION
**Pull request by lukicenturi**
https://docs.google.com/spreadsheets/d/1MNF6gsUmlZEhVbFg5f3ZzGSSJMu8Ti4aXx8KXSUL4j4/edit#gid=0&range=M350:M351

### CODE REVIEW

File: frontend/app/src/components/helper/PrioritizedList.vue [hunk 1]
```diff
@@ -103,59 +103,58 @@ const autoCompleteHint: ComputedRef<string> = computed(() => {
 
 <template>
   <div>
-    <VSheet outlined rounded>
-      <div class="pb-0" :class="slots.title ? 'pa-4' : 'pa-0'">
-        <div v-if="slots.title" class="text-h6 pb-4">
+    <RuiCard rounded="md" class="[&>div]:!p-0">
```
Comment (tewshi) [inline frontend/app/src/components/helper/PrioritizedList.vue:106]: we can probably add a prop to card to not add padding (some libs use `flush`).
doesn't affect your PR though.

File: frontend/app/src/components/helper/PrioritizedList.vue [hunk 2]
```diff
@@ -165,78 +164,63 @@ const autoCompleteHint: ComputedRef<string> = computed(() => {
...
-                    icon
-                    small
-                    :disabled="isLast(identifier)"
-                    @click="move(identifier, true)"
-                  >
-                    <VIcon>mdi-chevron-down</VIcon>
-                  </VBtn>
-                </div>
+                <RuiButtonGroup variant="outlined" size="sm" icon vertical>
+                  <template #default>
```
Comment (tewshi) [inline frontend/app/src/components/helper/PrioritizedList.vue:176]: just to be sure, do you need to put it in default slot this way?

File: frontend/app/src/components/helper/PrioritizedList.vue (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/components/helper/PrioritizedList.vue:106]: Yeah that sounds good. We can do that for next release

File: frontend/app/src/components/helper/PrioritizedList.vue (hunk 2 above)
Comment (lukicenturi) [reply frontend/app/src/components/helper/PrioritizedList.vue:176]: There is a bug with <ButtonGroup> and <RadioGroup>, the children aren't reactive.
We need to do this until we move to vue 3

**Review by tewshi — APPROVED** [review]

**Pull request merged**